modules/docs/arrow-docs/docs/docs/typeclasses/foldable/README.md

```diff
@@ -9,8 +9,518 @@ permalink: /docs/typeclasses/foldable/
 {:.intermediate}
 intermediate
 
-TODO. Meanwhile you can find a short description in the [intro to typeclasses]({{ '/docs/typeclasses/intro/' | relative_url }}).
+The Typeclass `Foldable` provide us the ability of, given a type `F<A>`, fold their values `<A>`.
```

Comment:
Let's be consistent with Arrow nomenclature for this, use `Kind<F, A>` instead of `F<A>`.

Reply:
Also, `fold` is used in the description but also in the name. It'd be worth explaining that fold means `collecting`, or `aggregating`.